server/odc-service/src/main/java/com/oceanbase/odc/service/session/OdcStatementCallBack.java

```diff
@@ -336,6 +345,45 @@ private List<JdbcGeneralResult> consumeStatement(Statement statement, SqlTuple s
         return executeResults;
     }
 
+    private void checkIsExternalTalbe(JdbcGeneralResult jdbcGeneralResult) {
```

Comment:
don't do this in the class, you may do this in `SqlExecuteResult#initEditableInfo`